app/models/chargeback_rate_detail.rb

```diff
@@ -71,15 +96,20 @@ def rate_name
   end
 
   def friendly_rate
+    find_rate(0.0)
```

Comment:
I don't understand why this is called.  Are we doing this for the side effect of setting the `@fixed_rate` and `@variable_rate`?